src/ex5/dist/clients/item_client.js

```diff
@@ -1,36 +1,45 @@
-
 class itemClient {
     
-    constructor(){
-        this.todoList = [];
-    }
-
     async getTodo() {
-       const response = await fetch("/getAll");
+       const response =await fetch("/getAll");
        return await response.json();
     }
 
-    async addTodo(todoList) {
-        const response=await fetch("/",{
+    async addTodo(todo) {
+        await fetch("/",{
             method: 'POST',
             headers: {'Content-Type': 'application/json'},
-            body: JSON.stringify(todoList)
+            body: JSON.stringify({todo})
         })
-       return await response.json();
-     }
+    }
 
-     async deleteTodo(id) {
-        const response=await fetch("/"+id,{
-            method: 'DELETE'
+    async deleteTodo(id) {
+        await fetch("/id",{
+            method: 'DELETE',
+            headers: { "Content-Type": "application/json" },
+            body: JSON.stringify({id})
         })
-       return await response.json();
-     }
+    }
 
-     async deleteAllTodo() {
-        const response=await fetch("/",{
+    async deleteAllTodo() {
+        await fetch("/",{
             method: 'DELETE'
         })
-       return await response.json();
-     }
- }
+    }
+
+    async changeDoneTodo(id,status) {
+        await fetch("/status",{
```

Comment:
there is some naming inconsistencies with your endpoints -
changing the 'done' field is performed by /status and changing the todo field is performed by /item
this could make tracing calls a bit confusing